books/RayTracingTheRestOfYourLife.html

```diff
@@ -197,17 +226,73 @@
 On my computer, I get:
 
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    Regular    Estimate of Pi = 3.14151480
-    Stratified Estimate of Pi = 3.14158948
+    Regular    Estimate of Pi = 3.141184000000
+    Stratified Estimate of Pi = 3.141460000000
```

Comment:
The reader has to mentally compare these values to their memorized value of π. I think we should consider adding a report of the error, to some fixed number of digits.

Reply:
Hmmmmmmmm. Maybe a %...
I have the first 6 digits of pi memorized (it's a cooool hobby). So I could immediately look at this and see the error. But see my concerns above about comparing directly to pi programmatically. My compromise here is to write out the value of pi, and let the student eyeball it.